src/Shared/PropertyFetcher.AOT.cs

```diff
@@ -9,10 +9,14 @@
 
 #nullable enable
 
+#pragma warning disable IDE0005 // Using directive is unnecessary.
 #if NETSTANDARD2_1_0_OR_GREATER || NET6_0_OR_GREATER
 using System.Diagnostics.CodeAnalysis;
 #endif
+using System;
+using System.Linq;
 using System.Reflection;
+#pragma warning restore IDE0005 // Using directive is unnecessary.
```

Comment:
Curious why do we need to do this?

Reply:
Part of the projects are using implicit usings. It is not enabled globally.